sp800-63c/sec6_presentation.md

```diff
@@ -18,16 +18,25 @@ In the *back-channel* model, the subscriber is given an assertion reference to p
 
 In this model, the assertion itself is requested directly from the IdP to the RP, minimizing chances of interception and manipulation by a third party (including the subscriber themselves). 
 
-This method also allows the RP to query the CSP for additional attributes about the subscriber not included in the assertion itself, as back-channel communication can occur after the initial authentication transaction. 
+This method also allows the RP to query the CSP for additional attributes about the subscriber not included in the assertion itself, as back-channel communication can occur after the initial authentication transaction, or when the subscriber is no longer available. 
```

Comment:
"when the subscriber is no longer available". I'm wondering whether this is in-scope for 800-63, or if at that point we're talking about a machine-to-machine, and not a user-to-machine transaction.